emperor/support_files/js/controller.js

```diff
@@ -302,6 +331,61 @@ define([
 
   /**
    *
+   * Write save file for the current controller settings
+   *
+   **/
+   EmperorController.prototype.saveConfig = function() {
+    var saveinfo = {};
+    // Assuming single sceneview for now
+    sceneview = this.sceneViews[0];
+    saveinfo.cameraPos = sceneview.camera.position;
+    saveinfo.cameraQuat = sceneview.camera.quaternion;
+    //Save settings for each controller in the view
+     _.each(this.controllers, function(controller, index) {
+      if (controller === undefined) {
+        return;
+      }
+      saveinfo[index] = controller.toJSON();
+    });
+
+    //save the file
+    var blob = new Blob([JSON.stringify(saveinfo)], {type: "text/plain;charset=utf-8"});
```

Comment:
Out of curiosity I looked this up, and I think this should be a different MIME i.e. `"application/json"`:
http://stackoverflow.com/a/477819